src/shared/sleep-config.c

```diff
@@ -415,6 +416,11 @@ int can_sleep(const char *verb) {
         if (is_product_listed(blacklist))
                 return false;
 
+        /* https://phabricator.endlessm.com/T20298 */
```

Comment:
I also agree that this change should be made in sleep-products.conf, not in the code.
There is no need to worry about autotools stuff since that file is already installed. Just add the entries to that file.

DMI product_name on the new bios is `EE-200`, not `F3C`.

Reply:
My explicit +1 to @jprvita and @dsd suggestion of not hard-coding it and simply adding the entry to the conf file. If that does not work, we'd have more serious problem to solve, really :)

Reply:
@dsd good catch. I guess I was accidentally looking at the `product_name` on a machine with the old BIOS